gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LaunchConfig.java

```diff
@@ -168,9 +168,10 @@ static void action(Params params) {
                         .addAllJvmOpts(avxOptions)
                         .addAllJvmOpts(params.getAddJava8GcLogging().get() ? java8gcLoggingOptions : ImmutableList.of())
                         // Java 11.0.16 introduced a potential memory leak issues when using the C2
-                        // compiler
+                        // compiler, resolved in 11.0.16.1
                         .addAllJvmOpts(
                                 javaVersion.compareTo(JavaVersion.toVersion("11")) == 0
+                                                && !params.getBundledJdks().get()
                                         ? jdk11DisableC2Compile
                                         : ImmutableList.of())
```

Comment:
I'm fine with PR as it is now, but wondering if we should completely remove the compile command even on JDK 11 as anything picking up new sls-packaging release would have dependency on fixed JDKs at this point.

```suggestion
```

Reply:
The failure mode is a potentially old depot, however it is unlikely